saskatoon/sitebase/templates/app/index.html

```diff
@@ -22,7 +22,11 @@ <h2>{% trans "Hi" %} {{user}} !</h2>
                                         <hr>
                                         {% include 'app/index/pickleader.html' %}
                                     {% else %}
-                                        <h2>{% trans "Welcome to Les Fruits Défendus!" %}</h2>
+                                        {% if homepage %}
```

Comment:
In case the user does not create entry in the homepage model, we should display the dafault welcome message.